extensions/amp-story/1.0/amp-story.css

```diff
@@ -12,6 +12,13 @@
 @import './pagination-buttons.css';
 
 /** Common */
+:root {
+  --story-page-vw: 1vw;
+  --story-page-vh: 1vh;
```

Comment:
Can we get these right for Desktop too, or is it too much work?

Reply:
I started drafting what it would take for this to work with desktop 1 panel on #35978 (not prod code) so it's possible. Will do that after this PR

Reply:
I'm a bit scared about the difference between vh and outerHeight across browsers tbh. I think it might be off when there are large browser bars, like for in-app browsers.
Can you try the same Story one pre PR, one post PR, and open it in the Hangout in-app browser, see if the layouts exactly match? Let's try both iOS and Android.

Reply:
Tested and there was no noticeable difference between CSS and JS variable values